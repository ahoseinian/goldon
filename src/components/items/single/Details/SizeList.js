import React from 'react';
import Row from './Row'

const SizeList = ({tool, arz, ertefa, ghotr}) => (
  <table className="table table-bordered mt-4">
    <tbody>
      <Row name="طول" value={tool} measure="سانتی متر"/>
      <Row name="عرض" value={arz} measure="سانتی متر"/>
      <Row name="ارتفاع" value={ertefa} measure="سانتی متر"/>
      <Row name="قطر" value={ghotr} measure="سانتی متر"/>
    </tbody>
  </table>
);

SizeList.propTypes = {
  tool: React.PropTypes.number,
  arz: React.PropTypes.number,
  ertefa: React.PropTypes.number,
  ghotr: React.PropTypes.number
};

export default SizeList;
