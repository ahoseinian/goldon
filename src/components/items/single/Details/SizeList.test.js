import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import SizeList from './SizeList'

const sizes = {
  tool: 1,
  arz: 2,
  ertefa: 3,
  ghotr: 4
}

describe('SizeList', () => {
  it('renders', () => {
    const wrapper = shallow(<SizeList {...sizes}/>)
    expect(toJson(
      wrapper
    )).toMatchSnapshot()
  })
})
