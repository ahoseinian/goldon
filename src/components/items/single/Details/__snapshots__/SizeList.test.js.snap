exports[`SizeList renders 1`] = `
<table
  className="table table-bordered mt-4">
  <tbody>
    <SingleRow
      measure="سانتی متر"
      name="طول"
      value={1} />
    <SingleRow
      measure="سانتی متر"
      name="عرض"
      value={2} />
    <SingleRow
      measure="سانتی متر"
      name="ارتفاع"
      value={3} />
    <SingleRow
      measure="سانتی متر"
      name="قطر"
      value={4} />
  </tbody>
</table>
`;
